app/src/forms/email/emailService.js

```diff
@@ -88,6 +97,53 @@ const service = {
     }
   },
 
+  // Assigning user to Submission Draft
+  submissionAssigned: async (formId, currentStatus, assignmentNotificationEmail, referer) => {
+    try {
+      const form = await formService.readForm(formId);
+      const submission = await formService.readSubmission(currentStatus.formSubmissionId);
+      const configData = {
+        bodyTemplate: 'submission-assigned.html',
+        title: `Invited to ${form.name} Draft`,
+        subject: 'Invited to Submission Draft',
+        messageLinkText: `You have been invited to a ${form.name} submission draft. You can review your submission draft details by visiting the following links:`,
+        priority: 'normal',
+        assignmentNotificationEmail,
+        form,
+      };
+      return service._sendEmailTemplate('sendSubmissionAssigned', configData, submission, referer);
+    } catch (e) {
+      log.error(`status: ${currentStatus}, referer: ${referer}`, { function: 'statusAssigned' });
+      log.error(e.message, { function: 'submissionAssigned' });
+      log.error(e);
```

Comment:
I'd strongly recommend utilizing more of winston's new logging features instead of multi-line reporting the same error repeatedly here. Winston logs are already output in structured JSON, so we should be taking advantage of that instead. Repeat for other similar sections. For example, you can try experimenting with the following to be more precise with your logging:
```js
log.error(e.message, {
  function: 'statusAssigned',
  status: currentStatus,
  referer: referer
});
```

Reply:
Changed the error logs accordingly for all the emailService functions.